templates/wikijs.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Services,
+  createTemplate,
+  randomPassword,
+} from "~templates-utils";
+
+export default createTemplate({
+  name: "Wiki.js",
+  schema: {
+    type: "object",
+    required: [
+      "projectName",
+      "domain",
+      "appServiceName",
+      "databaseType",
+      "databaseServiceName"
+    ],
+    properties: {
+      projectName: {
+        type: "string",
+        title: "Project Name",
+      },
+      domain: {
+        type: "string",
+        title: "Domain",
+      },
+      appServiceName: {
+        type: "string",
+        title: "App Service Name",
+        default: "wikijs",
+      },
+      databaseType: {
+        type: "string",
+        title: "Database Type",
+        default: "postgres",
+        oneOf: [
+          { enum: ["postgres"], title: "Postgres" },
+          { enum: ["mysql"], title: "MySQL" },
+        ],
+      },
+      databaseServiceName: {
+        type: "string",
+        title: "Database Service Name",
+        default: "db",
+      },
+    },
+  } as const,
+  generate({
+    projectName,
+    domain,
+    appServiceName,
+    databaseType,
+    databaseServiceName,
+  }) {
+    const services: Services = [];
+    const databasePassword = randomPassword();
+    const databaseUsername = databaseType === "postgres" ? "postgres" : "mysql";
+    const databasePort = databaseType === "postgres" ? "5432" : "3306";
+
+    services.push({
```

Comment:
I think you forgot to specify the service type

Reply:
Haha, Thanks, was a little too quick there.